source/appModules/hxmail.py

```diff
@@ -12,13 +12,32 @@
 
 class MailWordDocumentTreeInterceptor(WordDocument.treeInterceptorClass):
 
-	def _get_isAlive(self):
-		return super(MailWordDocumentTreeInterceptor,self).isAlive and self.rootNVDAObject.shouldCreateTreeInterceptor
+	_wasInReadingPane: bool = False

```

Comment:
Should this be initialized to False? Currently this will only be accurate after the gain focus event is fired the first time. Could this be initialized correctly in the `__init__` method instead.

Could a situation arise where the object is created while focus is already in a readingpane? Thus no initial fire of the gainfocus to set this to True, but rendering it inaccurate?